<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleAuxiliaryPage</class>
 <widget class="QWidget" name="VehicleAuxiliaryPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(0,0,50);
}
QLabel
{

font:24px,&quot;微软雅黑&quot;;
border:1px solid black;
}</string>
  </property>
  <widget class="QLabel" name="LBLTC1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>211</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLM2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>60</y>
     <width>211</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLM1">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>211</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLMP1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>211</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_AC1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>95</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/sivunknown2.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_DC1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>95</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/sivunknown.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_AC2">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>95</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/sivunknown2.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_DC2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>95</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/sivunknown.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_metroNum2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>25</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 170, 170);
font:20px,&quot;微软雅黑&quot;;
border:0px,black</string>
   </property>
   <property name="text">
    <string>MP1车</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_metroNum3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>25</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 170, 170);
font:20px,&quot;微软雅黑&quot;;
border:0px,black</string>
   </property>
   <property name="text">
    <string>MP2车</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_metroNum4">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>25</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 170, 170);
font:20px,&quot;微软雅黑&quot;;
border:0px,black</string>
   </property>
   <property name="text">
    <string>MC2车</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_metroNum1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>25</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 170, 170);
font:20px,&quot;微软雅黑&quot;;
border:0px,black</string>
   </property>
   <property name="text">
    <string>MC1车</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_metroPower1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>95</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/doorunknow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_metroPower2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>95</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/doorunknow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cab1_direction">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>105</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/arrowleft.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cab1_active">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>108</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 176, 80);
border-radius:8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cab2_direction">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>105</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/arrowright.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cab2_active">
   <property name="geometry">
    <rect>
     <x>941</x>
     <y>108</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 176, 80);
border-radius:8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>41</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border-top-left-radius:8px;
border-bottom-left-radius:8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>60</y>
     <width>41</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border-top-right-radius:8px;
border-bottom-right-radius:8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>LBLTC1</zorder>
  <zorder>LBLM2</zorder>
  <zorder>LBLM1</zorder>
  <zorder>LBLMP1</zorder>
  <zorder>lbl_AC1</zorder>
  <zorder>lbl_DC1</zorder>
  <zorder>lbl_AC2</zorder>
  <zorder>lbl_DC2</zorder>
  <zorder>lbl_metroNum2</zorder>
  <zorder>lbl_metroNum3</zorder>
  <zorder>lbl_metroNum4</zorder>
  <zorder>lbl_metroNum1</zorder>
  <zorder>lbl_metroPower1</zorder>
  <zorder>lbl_metroPower2</zorder>
  <zorder>lbl_cab1_direction</zorder>
  <zorder>lbl_cab1_active</zorder>
  <zorder>lbl_cab2_direction</zorder>
  <zorder>lbl_cab2_active</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
